### PR TITLE
Fluid distributed training benchmark

### LINKED ISSUE #7409: Fluid distributed training benchmark design

### PR DESCRIPTION
**Pull request by Yancey1989**
Fixed #7409

### CODE REVIEW

File: benchmark/cluster/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  - Linux Kernel: v3.10.0
+
+- Resource
+  - CPU: 10 Cores per Pod
+  - Memory: 5GB per Pod
+
+- Docker Image
+
+  We use different base Docker Image to run the benchmark on Kubernetes:
+  - PaddlePaddle v2: paddlepaddle/paddle:latest
```
Comment (typhoonzero) [inline benchmark/cluster/README.md]: Should use a static tag, so when `latest` tag updates, this benchmark still can be reproduced.

File: benchmark/cluster/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  - PaddlePaddle Fluid: paddlepaddle/paddle:latest
+  - TensorFlow: tensorflow/tensorflow:latest
+
+- Model
+  A digits recognize model and MNIST dataset is used in this benchmark.
```
Comment (helinwang) [inline benchmark/cluster/README.md]: I think this model is too small. Maybe [vgg-16](https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/v2/fluid/tests/book/test_image_classification_train.py#L90) (probably around 500MB) is a better choice.

File: benchmark/cluster/README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+## Compare the Performance
+
+- Variable
+  - Batch Size of training data.
+  - PServer count of the training job.
+
+- Invariant
+  - The number of trainers.
```
Comment (helinwang) [inline benchmark/cluster/README.md]: What is the trainer count we plan to try?

File: benchmark/cluster/README.md (hunk 1 above)
Comment (Yancey1989) [reply benchmark/cluster/README.md]: Sure, but we don't have a static tag for fluid distributed training, how about a commit ID?

File: benchmark/cluster/README.md (hunk 2 above)
Comment (Yancey1989) [reply benchmark/cluster/README.md]: Done.

File: benchmark/cluster/README.md (hunk 3 above)
Comment (Yancey1989) [reply benchmark/cluster/README.md]: Done. 
And @typhoonzero reminds me that we need to measure the parallel efficiency by increasing the trainer count.

**Comment by Yancey1989** [comment]
From @typhoonzero 
> Should we put this doc into design or a separated repo?

Maybe not, I saw https://github.com/dzhwinter/benchmark is working for the Fluid benchmark, and I knew from @dzhwinter that will be merged into the Paddle repo in this week.

File: benchmark/cluster/README.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  - Linux Kernel: v3.10.0
+
+- Resource
+  - CPU: 10 Cores per Pod
+  - Memory: 5GB per Pod
+
+- Docker Image
+
+  We use different base Docker Image to run the benchmark on Kubernetes:
+  - PaddlePaddle v2: paddlepaddle/paddle:[commit-id]
```
Comment (typhoonzero) [inline benchmark/cluster/README.md]: v2 should use 0.10.0 tag, and fluid should use commit id

Comment (Yancey1989) [reply benchmark/cluster/README.md]: Done, since 0.10.0 does not support distributed training, use 0.11.0 .

**Pull request merged**